mods/default/craftitems.lua

```diff
@@ -160,8 +160,10 @@ minetest.register_craftitem("default:skeleton_key", {
 			return itemstack
 		end
 
-		local on_skeleton_key_use = minetest.registered_nodes[node.name].on_skeleton_key_use
-		if not on_skeleton_key_use then
+		local on_skeleton_key_use
+		if minetest.registered_nodes[node.name] and minetest.registered_nodes[node.name].on_skeleton_key_use then
```

Comment:
It would be better to factor out `minetest.registered_nodes[node.name]` to a local variable here

Reply:
Addressed

Reply:
woops and once more for good measure (I accidentally used spaces instead of tabs)